### DISCUSSION TITLE
Insert the pcb into the petling

### DISCUSSION THREAD

**Discussion by gewin1988** [universam1/iSpindel#568] [Q&A]
I've found a pcb months ago, and i've ordered it. I was in a hurry and i did not look after the relevant forums, so this is a kinda' "old" pcb. (the project was on hold until last week) I have all of the necessary equipment with the following tube: 
https://www.ispindel.site/.../xl-petling-tube-for-the.../
and the pcb :https://ibb.co/NrCpgQV
The newer pcb versions (4.0) corrected some failures from this one (i guess its v2). 
My problem is, that the pcb is not fit in the petling. What is the trick? I thought a newer pcb can solve this issue, but based on the pcb model's description the new version is wider 😕
(its not fit because if a battery holder added to the board (item based on iSpindel's description), the pcb can not go into the petling at it's diameter)
Is there any trick to fit the iSpindel into the petling? (I've carefully tried a rasp, but it not worked, maybe i should try again more agressively?)

**Comment by ErikdBr** [comment]
As far as I can see you have the same petling as me and allthough I use a different PCB (The Jeffrey V2.69) that should make no difference regarding the battery holder. I also found this on ebay which uses the newer variant of the Cherry Philip PCB which look like the same petling: https://www.ebay.co.uk/itm/193734736655?hash=item2d1b7d7b0f:g:QHsAAOSwdytfoHIL
So maybe it is your battery holder that is different?
My iSpindel:
![20220127_110638](https://user-images.githubusercontent.com/48140503/151339276-c46d782b-0a1e-4191-8c45-a6118ae9f71f.jpg)
